heroic-core/src/main/java/com/spotify/heroic/http/HttpServer.java

```diff
@@ -230,7 +230,7 @@ private HandlerCollection setupHandler() throws Exception {
         return handlers;
     }
 
-    private void makeRewriteRules(RewriteHandler rewrite) {
+    private static void makeRewriteRules(RewriteHandler rewrite) {
```

Comment:
Why is this static?

Reply:
because static methods are easier to test, easier to reason about and faster to execute.